src/test/java/org/apache/commons/text/similarity/IntersectionResultTest.java

```diff
@@ -45,6 +45,15 @@ void testEquals() {
             for (int j = 0; j < results.length; j++) {
                 Assertions.assertEquals(results[i].equals(results[j]), i == j);
             }
+
+            // IntelliJ would like to optimize this, but it would make
+            // the test useless as assertNotEquals() handles null itself
+            //noinspection ConstantValue,SimplifiableAssertion
+            Assertions.assertFalse(results[i].equals(null),
```

Comment:
No newspaper column-style formatting, please. See our checkstyle.xml file, which allows line lengths up to 160.